lib/db_helper/tasks.rb

```diff
@@ -176,9 +209,11 @@
 
       selection_proc = ->(sandbox) { "#{sandbox.foreground(:cyan)}: #{ports[sandbox].foreground(:green)}" }
       sandboxes = ports.keys.sort_by { |sandbox| ports[sandbox] }
-      selected_sandbox = InputReader.select_item(sandboxes,
-                                                 selection_attribute: selection_proc,
-                                                 prompt: 'Which sandbox port would you like to change?')
+      selected_sandbox =
+        InputReader.select_item(
+          sandboxes,
+          selection_attribute: selection_proc,
+          prompt: 'Which sandbox port would you like to change?')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.